### PR TITLE
[SITES-520] - update homepage category descriptions

### PR DESCRIPTION
**Pull request by simonschwartz**

### CODE REVIEW

**Comment by ltankey** [comment]
Just needs a changelog update

File: config/static_data.yml [hunk 1]
```diff
@@ -1,70 +1,56 @@
...
   - name: Immigration, visas and citizenship
     summary: >
-      Australian citizenship, customs, work, study and short-term visas,
-      migration and tourism.
+      Australian citizenship, customs, visas, migration and tourism.
   - name: Infrastructure and telecommunications
     summary: >
-      Regional development, transport safety, Television (TV) reception,
-      internet, telecommunications.
+      Transport safety, Television (TV) reception, telecommunications.
```
Comment (tessereth) [inline config/static_data.yml]: Possibly shouldn't have the capital letter on Television

File: config/static_data.yml [hunk 2]
```diff
@@ -1,70 +1,56 @@
...
   - name: About Australia
     summary: >
-      Intergenerational report, currency, National archives collection,
-      Australian honours, the constitution.
+      Currency, national archives, Australian honours, the constitution.
   - name: About Government
     summary: >
-      Government structure, freedom of information, elections and voting,
-      Budget, Australian Public Service.
+      Government structure, elections, Budget, Australian Public Service.
```
Comment (tessereth) [inline config/static_data.yml]: Possibly shouldn't have the capital letter on Budget

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,8 @@ This file is influenced by http://keepachangelog.com/.
 
 - Updated 'Popular now' links
 
+- Updated homepage category descriptions
```
Comment (tessereth) [inline CHANGELOG.md]: There shouldn't be a blank line between this and the previous dot point (I know that's how the others look but it still shouldn't have the blank line)

File: config/static_data.yml [hunk 4]
```diff
@@ -1,70 +1,56 @@
...
   - name: Education
     summary: >
-      Early childhood, school, higher education, international students,
-      skills recognition, scholarships, VET.
+      Early childhood, school, higher education, skills recognition, VET.
   - name: Emergencies and disasters
     summary: >
-      Triple Zero (000), natural disasters, search and rescue, disaster
-      assistance, health emergencies.
+      Triple Zero (000), natural disasters, health emergencies.
```
Comment (bjgordon) [inline config/static_data.yml]: Capital **Z**ero?

**Comment by ltankey** [comment]
LGTM

**Pull request merged**